proto/v1alpha1/rode.proto

```diff
@@ -51,9 +51,9 @@ service Rode {
       get: "/v1alpha1/generic-resources"
     };
   }
-  rpc ListOccurrences(ListOccurrencesRequest) returns (ListOccurrencesResponse) {
+  rpc ListVersionedResourceOccurrences(ListVersionedResourceOccurrencesRequest) returns (ListVersionedResourceOccurrencesResponse) {
     option (google.api.http) = {
-      get: "/v1alpha1/occurrences"
+      get: "/v1alpha1/versioned-resource-occurrences"
```

Comment:
For the path here, I was originally thinking `/v1alpha1/versioned-resources/{resource_uri=**}/occurrences`. This almost works, but unfortunately `grpc-gateway` does its routing on the unescaped URL path, so any percent-encoded slashes in the path are the treated the same as if they were unencoded originally. Setting `{resource_uri=**}` allows for capturing as many path segments as we like, but doesn't play well with schemes. For instance, `git://foo` becomes `git:/foo` by the time it reaches our handler.

We could get around this by double percent-encoding the resource uri or always passing it as a query parameter. It makes the most sense to pass in the path, but having to double-encoding is awkward.

Reply:
Related issue and closed pr:

```
https://github.com/grpc-ecosystem/grpc-gateway/issues/308
https://github.com/grpc-ecosystem/grpc-gateway/pull/660
```